statics/js/components/graph-layout.js

```diff
@@ -94,21 +94,62 @@ var LinkLabelLatency = Vue.extend({
 
     flowQuery: function(nodeTID, trackingID, limit) {
       let query = `G.Flows()`;
-      query += `.Has("NodeTID", "${nodeTID}")`;
+      query += `.Has("NodeTID", ${nodeTID})`;
       if (typeof trackingID !== 'undefined') {
-        query += `.Has("TrackingID", "${trackingID}")`;
+        query += `.Has("TrackingID", ${trackingID})`;
```

Comment:
Would be better to use one Has in term of performances

Reply:
ok, using one 'Has(...)'